pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -687,20 +639,10 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, _ v1.PodStatus, podStat
 
 		// Successfully started the container; clear the entry in the failure
 		glog.V(4).Infof("Completed init container %q for pod %q", container.Name, format.Pod(pod))
-		return
-	}
-	if !done {
-		// init container still running
-		glog.V(4).Infof("An init container is still running in pod %v", format.Pod(pod))
-		return
```

Comment:
Why this is removed? kubelet should wait all init containers finished before starting regular containers.